src/graphql/mutations/user.graphql

```diff
@@ -1,5 +1,17 @@
 mutation CreateUniqueUser($input: CreateUniqueUserInput!) {
-    createUniqueUser(input: $input) {
-        id
-    }
+  createUniqueUser(input: $input) {
+      id
+  }
+}
+
+mutation UpdateUserProfile ($input: UpdateProfileInput!) {
+  updateProfile(input: $input) {
+    id
+    avatar
+    bio
+    displayName
+    location
+    website
+    email
+  }
```

Comment:
I noticed that the thumbnail is not getting updated when uploading a new profile picture. Should the thumbnail be added here to the UpdateUserProfile mutation?

Reply:
Shouldn't we have some functionality for creating thumbnails? As it's outside of this PR I left it out. It seems that it's not worth to just store the same data in thumbnail as in avatar...